### PR TITLE
Alert box tweaks

### PR DESCRIPTION
**Pull request by PBruk**
@matt-bernhardt, please review. These commits include changes to menu; breadcrumbs; alert box; padding for Title and Main Content sections; with additional details and adjustments as tweaks were made.

### CODE REVIEW

**Comment by matt-bernhardt** [comment]
There's a lot to unpack in this. I'm in the midst of reviewing the changes, but one thing I think will help would be if there's a list of reference pages you used to judge these changes?

Part of the reason why I'm asking is because I'd like to try the regression tool I sometimes use - so if there's a list of URLs - and page elements - that I could feed it, that may help. Something like this:

```
example.org/foo/bar
#sidebar
#menu
.stage .breadcrumb

example.org/foo/baz
#header
#footer
```

**Comment by matt-bernhardt** [comment]
For the time being, I've chosen these three URLs on the dev server as test cases:
```
/
/barker
/hours
```

If there are better options, please let me know.

File: inc/nav-main.php [hunk 1]
```diff
@@ -138,6 +138,6 @@
...
+    <a href="/barton-account" class="no-underline account-link"><svg version="1.1" xmlns="http://www.w3.org/2000/svg" x="0" y="0" width="15.4" height="16" viewBox="0 0 15.4 16" enable-background="new 0 0 15.445 16" xml:space="preserve"><path d="M13.4 15.7C12.2 15.9 10.4 16 7.7 16c-5.4 0-7.3-0.6-7.3-0.6 -0.3-0.1-0.4-0.4-0.4-0.7 0.3-1.6 1.2-2.5 2.5-3.3 0.3-0.2 0.8-0.4 1.2-0.6 0.8-0.3 1.8-0.7 2-1.3C5.8 9.2 5.7 8.6 5.2 7.9c-1.4-2.3-1.7-4.3-0.8-5.9C5.1 0.7 6.4 0 7.7 0c1.4 0 2.6 0.7 3.3 2 0.9 1.6 0.7 3.6-0.8 5.9C9.8 8.6 9.6 9.2 9.8 9.6c0.2 0.6 1.2 1 2 1.3 0.4 0.2 0.9 0.4 1.2 0.6 1.2 0.8 2.1 1.6 2.5 3.3 0.1 0.3-0.1 0.6-0.4 0.7C15 15.4 14.5 15.6 13.4 15.7"/></svg><span>Account</span></a>
```
Comment (matt-bernhardt) [inline inc/nav-main.php]: Given the other changes in this PR, we probably need to look at removing this inline SVG in favor of http://fortawesome.github.io/Font-Awesome/icon/user/

**Comment by frrrances** [comment]
Visually reviewing: 
- [ ] menu
- [ ] breadcrumbs 
- [ ] alert box
- [ ] padding for Title and Main Content sections
- [ ] responsive for locations and study spaces

**Comment by frrrances** [comment]
All items in the main nav menu now stay in their boxes in Safari.

**Comment by frrrances** [comment]
Padding looks good on the main page title, content area and breadcrumbs on the location pages but the other pages (like About or Research support) don't look right - the breadcrumb has double padding and the title and content aren't lined up.

**Comment by matt-bernhardt** [comment]
I'm largely leaving the CSS style changes to @frrrances - which leaves looking over the following PHP templates:
- [ ] content-location.php
- [ ] inc/nav-main.php
- [ ] page-hours-json.php
- [ ] page-location.php
- [ ] page-main-locations.php
- [ ] page.php

Additionally, this javascript file is changed:
- [ ] js/alerts.js

**Comment by matt-bernhardt** [comment]
For future reference, this PR would be much easier to make sense of if the commits were composed differently. Right now it is difficult to tell what changes are releated to which issues - in part because some work seems to be unrelated to issues, but also because some apparently-related changes are spread out across multiple commits.

For example, the changes in commit 232842f add two rules to `css/partials/_alerts.scss`, but then two commits later in e77ff2e those rules are then removed from `css/partials/_page-titles.scss`. Ideally those two changes should be in a single commit, since the change is really to take rules from one location and move them elsewhere.

Additionally, commit e77ff2e has other changes as well - so if the reviewer is stepping through the request commit-by-commit, this grouping raises the possibility that these changes are somehow related.

**Comment by matt-bernhardt** [comment]
By contrast, commit a650c99 is better in some ways. There's a change from `#breadcrumb` to `.breadcrumbs` in the stylesheet, and then associated markup changes across a handful of templates. One conceptual change, to many files, that is easy to review.

However, in this commit there are also two other changes that _seem_ unrelated. It isn't clear that the change to .betterBreadcrumbs padding in `css/partials/_breadcrumbs.scss` applies to the changed markup, unless something is adding that rule to the markup at runtime?

This commit also includes some changes to `page-location.php` that have to do with restructuring of the location template (lines 37-47) that would be better grouped with other changes - for example dc2efbb which removes the div#stage from `content-location.php`

**Comment by matt-bernhardt** [comment]
Commit 2ecca45 is exactly what it should be, btw. Questions about which SVG needs to be inlined aside, the changes are grouped appropriately, with nothing extraneous that confuses the review.

File: page-hours-json.php [hunk 2]
```diff
@@ -137,7 +142,7 @@
 </script> 
 </div>
 <!--stage ends-->
-<div id="content" class="content-main">
+<div id="content" class="content-hours">
```
Comment (matt-bernhardt) [inline page-hours-json.php]: This is a change about which I'd like to hear more.

**Comment by matt-bernhardt** [comment]
As this branch didn't make it into `1.3.0`, I'm closing it. Please re-open a PR against `1.4.0` and we'll continue the review there.

**Pull request closed without merging**